geometry_msgs/Point pose
HumanBodyPart[] body_parts